src/components/SearchInput/index.js

```diff
@@ -19,7 +19,7 @@ class SearchInput extends Component {
   }
 
   render() {
-    const { handleSubmit } = this.props;
+    const { handleSubmit, buttonStyle = 'secondary' } = this.props;
```

Comment:
It might be good to use React's `defaultProps` for the `buttonStyle` property https://reactjs.org/docs/typechecking-with-proptypes.html#default-prop-values